sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/CHANGELOG.md

```diff
@@ -1,15 +1,42 @@
 # Release History
 
-## 1.2.0-beta.4 (Unreleased)
+## 1.2.0-beta.4 (2024-05-20)
 
 ### Features Added
 
-### Breaking Changes
+* Added CustomProperties to LiveMetrics Documents
+  ([#43600](https://github.com/Azure/azure-sdk-for-net/pull/43600))
 
 ### Bugs Fixed
 
+* Fixed a bug in LiveMetrics Document filtering
+  ([#43546](https://github.com/Azure/azure-sdk-for-net/pull/43546))
+
+### Other Changes
+
+* Update OpenTelemetry dependencies.
```

Comment:
Should we repeat this section? It is covered in 1.1.1, which is included here.

Reply:
Ah, I think it's safe to remove now. 

I thought it was important when the Changelog was in release order (ie: betas 1, 2, and 3 didn't have this change). With the new sort, I think it's fine to remove.